kyuubi-server/src/main/scala/org/apache/kyuubi/operation/BatchJobSubmission.scala

```diff
@@ -267,6 +269,10 @@ class BatchJobSubmission(
       }
     } finally {
       builder.close()
+      if (session.isResourceFromUpload) {
+        info(s"cleanup uploaded resource file after execution: $resource")
+        FileUtils.deleteQuietly(Paths.get(resource).toFile)
```

Comment:
It is better to provide more info.
```
if (!(FileUtils.deleteQuietly(Paths.get(resource).toFile)) {
  error(...)
}
```

Reply:
Addressed. Delete file and logging error with added `Utils.deleteFile` method.